File: 배열/4344/4344.py
c = int(input())
for i in range(c):
   score = list(map(int, input().split()))
   avg = sum(score[1:]) / score[0]
   ans = 0
   
   for j in range(1, len(score)):
       if score[j] > avg:
           ans += 1
   ans = ans / score[0] * 100
   print(format(ans, ".3f") + "%")

File: 배열/4344/4344.c
#include <stdio.h>
#define MAX 1000

int main()
{
   int c, n, i, j;
   float avg, ans;
   int score[MAX];

   scanf("%d", &c);
   for (i = 0; i < c; ++i)
   {
      avg = 0.f, ans = 0.f;
      scanf("%d", &n);
      for (j = 0; j < n; ++j)
      {
         scanf("%d", score + j);
         avg += score[j];
      }
      avg /= n;

      for (j = 0; j < n; ++j)
         if (score[j] > avg)
            ans += 1.f;
      ans = ans / n * 100;

      printf("%.3f%%\n", ans);
   }

   return 0;
}